hypothesis-python/src/hypothesis/strategies/_internal/types.py

```diff
@@ -256,9 +256,27 @@ def is_a_union(thing):
     )
 
 
+try:
+    import numpy as np
+except ImportError:
+
+    def is_np_type(thing):
+        return False
+
+else:
+
+    def is_np_type(thing):
+        return isinstance(thing, (np.ndarray, np.generic))
```

Comment:
```suggestion
def is_np_type(thing):
    np = sys.modules.get("numpy")
    if np is None:
        return False
    return isinstance(thing, (np.ndarray, np.generic))
```

Useful little trick which avoids importing Numpy when it's installed but not actually used by user code 😁